src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,19 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, entity, entityToEdit) => (values) => {
   const errors = {};
-
   _.forEach(fieldConfig, (config) => {
     const val = values[config.name];
-
-    if (config.required && (_.isNil(val) || val === '')) {
-      errors[config.name] = `${config.displayName} is required`;
-    }
+    const currentValue = entityToEdit ? entityToEdit[config.name] : null;
+    _.forEach(entity, (elem) => {
```

Comment:
You should think about the performance, this for is unnecessary, first you should check if the field is unique and only when it is you should look for entity with the same value, you can use `_.find` for this